sql/db_test.go

```diff
@@ -51,21 +53,25 @@ func TestMain(m *testing.M) {
 		}
 		testDB, e = Open(fmt.Sprintf("mysql://%s", cfg.FormatDSN()))
 		assertNoErr(e)
-		// Test for create database sql execution through DB driver.
+		defer testDB.Close()
 		_, e = testDB.Exec("CREATE DATABASE IF NOT EXISTS sqlflow_models;")
 		assertNoErr(e)
-		defer testDB.Close()
+		assertNoErr(testdata.Popularize(testDB.DB, testdata.IrisSQL))
```

Comment:
maybe use subtests like in :https://github.com/sql-machine-learning/sqlflow/blob/develop/cmd/sqlflowserver/main_test.go#L174